query-engine/driver-adapters/js/smoke-test-js/.envrc.example

```diff
@@ -1,5 +1,5 @@
 export JS_PLANETSCALE_DATABASE_URL="mysql://USER:PASSWORD@aws.connect.psdb.cloud/DATABASE?sslaccept=strict"
 export JS_NEON_DATABASE_URL="postgres://USER:PASSWORD@DATABASE-pooler.eu-central-1.aws.neon.tech/neondb?pgbouncer=true&connect_timeout=10"
 
-# Note: if you use hosted Postgres instances (e.g., from PDP provision), you need `?sslmode=disable`
+# Note: if you use hosted Postgres instances (e.g., from PDP provision), you need `?pgbouncer=true&sslmode=disable`
```

Comment:
That is only relevant if the PostgreSQL instance is behind PgBouncer. Otherwise this will just make things slower.

Reply:
Only added that to make a test easier :P

Removed the need for it by adjusting the test.